modules/unsupported/mbtiles/src/main/java/org/geotools/mbtiles/MBTilesMetadata.java

```diff
@@ -1,17 +1,48 @@
 package org.geotools.mbtiles;
+import java.util.Arrays;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 import org.geotools.geometry.jts.ReferencedEnvelope;
+import org.geotools.referencing.CRS;
+import org.geotools.util.logging.Logging;
 import org.opengis.geometry.Envelope;
+import org.opengis.geometry.MismatchedDimensionException;
+import org.opengis.referencing.FactoryException;
 
 public class MBTilesMetadata {
     
+    private static Logger LOGGER = Logging.getLogger(MBTilesMetadata.class); 
+    
     protected static Pattern patternEnvelope = Pattern.compile(" *(\\-?[0-9\\.]*) *, *(\\-?[0-9\\.]*) *, *(\\-?[0-9\\.]*) *, *(\\-?[0-9\\.]*) *");
     
-    public enum t_type {OVERLAY, BASE_LAYER} ;
+    public enum t_type {
+        OVERLAY("overlay"), 
+        BASE_LAYER("baselayer");
```

Comment:
That was the problem I was fixing.  The original code used an underscore for both which violated the spec.  I changed the string used in the file to not have an underscore (complying with the spec) but left the enum identifier as it was to avoid breaking anything.  During parsing I make it accept either string so that files written with older versions of GeoTools that did this wrong will still work.